.github/workflows/changelog_to_tag.yml

```diff
@@ -68,6 +81,77 @@ jobs:
           fi
           echo "tagname=$_tagname" >> "$GITHUB_OUTPUT"
           echo "branch=$_branch" >> "$GITHUB_OUTPUT"
+          popd
+
+      - name: Build and publish the collection
```

Comment:
Some of this stuff is done by `release_collection.py` - is there some reason you cannot use `release_collection.py` here?

Reply:
release_collection.py required collection_release.yml, I thought it is too specific to be used outside linux-system-roles and tried to do the same but in less steps.

Reply:
Looking at https://github.com/linux-system-roles/auto-maintenance/blob/main/collection_release.yml, Check the collection using `galaxy-importer` is a step that I miss here. I'll take a look at using this script once again.

Reply:
> release_collection.py required collection_release.yml, I thought it is too specific to be used outside linux-system-roles and tried to do the same but in less steps.

Ah, right, needs that file with a list of roles to checkout.  Hmm - not sure if you can use release_collection.py then

Reply:
release_collection.py has too many things related to linux-system-roles collection hard coded. There is not much code to reuse, I prefer it like it is now